server/adapters/postgres.go

```diff
@@ -689,6 +720,39 @@ func (p *Postgres) buildQueryPayload(valuesMap map[string]interface{}) (string,
 	return strings.Join(header, ", "), strings.Join(placeholders, ", "), values
 }
 
+func (p *Postgres) ValidateWritePermission() error {
```

Comment:
After moving test eventContext creation outside storage, this func can be omitted.

Reply:
Function was removed